src/modules/flight_mode_manager/tasks/AutoFollowTarget/follow_target_params.c

```diff
@@ -106,39 +108,22 @@ PARAM_DEFINE_FLOAT(NAV_FT_RS, 0.1f);
  * the target's altitude, the follow altitude NAV_MIN_FT_HT should be high enough
  * to prevent terrain collisions due to GPS inaccuracies of the target.
  *
- * TODO: Add option for 2D tracking + terrain following
- *
- * @value 0 Maintain constant altitude and track XY position only (2D tracking)
- * @value 1 Track target's altitude (3D tracking)
+ * @value 0 2D Tracking : Maintain constant altitude relative to home and track XY position only
+ * @value 1 2D + Terrain : Mantain constant altitude relative to terrain below and track XY position
+ * @value 2 3D Tracking : Track target's altitude (be aware that GPS altitude bias usually makes this useless)
  * @group Follow target
  */
-PARAM_DEFINE_INT32(NAV_FT_ALT_M, 0);
+PARAM_DEFINE_INT32(FLW_TGT_ALT_M, 0);
 
 /**
- * Gimbal tracking mode
+ * Maximum velocity setting for generating the follow orbit trajectory
  *
- * @value 0 2D tracking: Point at target XY coordinates, and at ground Z coordinate
- * @value 1 2D tracking with terrain: Point at target XY coordinates, and at terrain Z coordinate
- * @value 2 3D tracking: Point at target XYZ coordinates
- * @group Follow target
- */
-PARAM_DEFINE_INT32(NAV_FT_GMB_M, 0);
-
-/**
- * Compensate filter delay
- *
- * The follow flight mode is estimating and filtering the target's position
- * and velocity in order to achieve a smoother tracking behavior. The filtering
- * introduces some delay which is noticable when the target is moving at higher
- * speeds, such as the case for bicycles and cars. For this high-speed scenario
- * the delay compensation can be enabled, which extrapolates the filtered position
- * in order to compensate for the filter delay.
- * This setting should not be used when the target is moving slowly, for example
- * when it's a pedestrian, as it will then emphasize the measurement noise instead
- * and cause unnecessary movement for the drone.
- *
- * @value 0 disabled
- * @value 1 enabled
+ * This is the maximum velocity the drone will circle around the target whenever
```

Comment:
This is the "**tangential** velocity" on the circle then, right? Not angular velocity.